200_Language_intro/40_One_language_per_doc.asciidoc

```diff
@@ -1,10 +1,10 @@
 [[one-lang-docs]]
 === One Language per Document
 
-A single predominant language per document ((("languages", "one language per document")))((("indices", "documents in different languages")))requires a relatively simple setup.
+A single predominant language per document requires a relatively simple setup.
 Documents from different languages can be stored in separate indices&#x2014;`blogs-en`,
```

Comment:
Let's clean up these `&#x2014;` characters as well.  Another artifact from O'Reilly, which unfortunately breaks AsciiDoctor because they don't support arbitrary character codes (see: http://discuss.asciidoctor.org/Can-I-build-this-https-github-com-elastic-elasticsearch-definitive-guide-into-pdf-td3845.html#a3854)

Em-dash is supported natively with "space dash-dash space" (` -- `).  Complete set of out of the box codes here: http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#attributes-and-substitutions

If we run into any non-predefined codes we can make some custom attributes.  I'll send a mail about this for everyone else, forgot about this issue until now. :)